src/utils/useIdlAccount.ts

```diff
@@ -0,0 +1,26 @@
+import { BorshAccountsCoder, Idl, IdlAccounts } from '@project-serum/anchor'
+import { PublicKey } from '@solana/web3.js'
+import { useMemo } from 'react'
+import { TypedAccountParser } from '@helium/spl-utils'
+import { UseAccountState, useAccount } from './useAccount'
+import * as Logger from './logger'
+
+export function useIdlAccount<IDL extends Idl, A extends string = string>(
```

Comment:
so this is like a typed Account? I'm confused. lol

Reply:
useIdlAccount can get any anchor account data, typed using the IDL you pass in